service/src/main/java/com/codeforcommunity/processor/MapProcessorImpl.java

```diff
@@ -105,6 +107,22 @@ private NeighborhoodFeature neighborhoodFeatureFromRecord(
     }
   }
 
+  private SiteFeature siteFeatureFromRecord(
```

Comment:
Could you reply to this with a very short explanation of what this does?

Reply:
Converts a retrieved record from database table `site_entries` (joined with relevant columns from `sites` and `users`/`entry_usernames`) into a `SiteFeature` (a DTO class)